keyvault/2016-10-01/swagger/keyvault.json

```diff
@@ -2547,7 +2547,7 @@
           "type": "array",
           "items": {
             "type": "string",
-            "description": "JSON web key operations. For more information, see JsonWebKeyOperation.",
+            "description": "JSON web key operations. The default is all of the operation values; encrypt, decrypt, sign, verify, wrapKey and unwrapKey. For more information, see JsonWebKeyOperation.",
```

Comment:
According to the dev, @pomortaz, the default is all of the values. If there's a better way to represent, that fits the general docs case of swagger, I'm interested.